src/pages/CreateCb.js

```diff
@@ -0,0 +1,232 @@
+import Typography from '@material-ui/core/Typography';
+import Paper from '@material-ui/core/Paper';
+import SnackbarContent from '@material-ui/core/SnackbarContent';
+import TextField from '@material-ui/core/TextField';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import InputLabel from '@material-ui/core/InputLabel';
+import FormControl from '@material-ui/core/FormControl';
+import Button from '@material-ui/core/Button';
+
+import { api } from '../utils/api'
+import { StatusEnum } from '../utils/enums';
+
+const styles = theme => ({
+  container: {
+    width: '100%',
+    marginTop: 20,
+    overflowX: 'auto',
+    padding: 5,
+  },
+  textField: {
+    marginLeft: theme.spacing.unit,
+    marginRight: theme.spacing.unit,
+    width: 200,
+  },
+  dense: {
+    marginTop: 19,
+  },
+  menu: {
+    width: 200,
+  },
+  snackbar: {
+    margin: theme.spacing.unit,
+  },
+  selectEmpty: {
+    marginTop: theme.spacing.unit * 2,
+  },
+  formControl: {
+    margin: theme.spacing.unit,
+    minWidth: 120,
+  },
+});
+
+const inlinePaperStyle = {
+  marginTop: 20,
+}
+
+const turnoverBands = ['<£100k', '£100k-£250k', '£250k-£500k', '£500k-£750k', '£750k-£1m', '£1m-£5m', '£5m-£10m', '>£10m']
+
+class OrgProfile extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      status: StatusEnum.PENDING,
+      formUpdate: {},
+      message: null,
+      regions: [],
+      sectors: [],
+    };
+  }
+
+  componentDidMount() {
+    Promise.all([api.regions(), api.sectors()])
+      .then(([regionsRes, sectorsRes]) => {
+        this.setState({
+          regions: regionsRes.data.result,
+          sectors: sectorsRes.data.result,
+          status: StatusEnum.SUCCESS
+        })
+      })
+      .catch((err) => {
+        const errorMessage = (err.response && err.response.data.error.message) || 'Error fetching data.'
+        this.setState({ message: errorMessage })
+      })
+  }
+
+  handleChange = name => event => {
+    this.setState({
+      formUpdate: {
+        ...this.state.formUpdate,
+        [name]: event.target.value
+      },
+    })
+  };
+
+  submitForm = () => {
+    const { formUpdate } = this.state;
+    api.register({ formUpdate })
+      .then(() => {
+        this.setState({
+          message: 'CB has been created. An invite email has been sent to the email provided.'
+        })
+      })
+      .catch((err) => {
+        this.setState({ message: err.response.data.error.message })
+      })
+  }
+
+  render() {
+    const {
+      props: { classes },
+      state: { message, formUpdate, regions, sectors }, handleChange, submitForm } = this
+
+    return (
+      <Fragment>
+        <Typography variant="h5" component="h3">
+          Create New Community Business
+        </Typography>
+        <Paper className={classes.container}>
+          <form className={classes.container} noValidate autoComplete="off">
```

Comment:
We should add `required` on at least name, 360Id, region and sector?

Reply:
We do not require a 360 giving id for an cb/organisation. I'll add it on the others.